corehq/apps/hqwebapp/static/hqwebapp/js/initial_page_data.js

```diff
@@ -89,7 +89,7 @@ hqDefine('hqwebapp/js/initial_page_data', ['jquery', 'underscore'], function ($,
         var args = arguments;
         var index = 1;
         if (!urls[name]) {
-            urls = gather(url_selector, urls);
+            _.extend(urls, gather(url_selector, urls));
```

Comment:
Will it matter if `urls` contains duplicate items? Given the code this is replacing, I'm assuming that `gather(url_selector, urls)` will return everything that is already in `urls` before plus anything new.

Reply:
`urls` shouldn't contain duplicate items. Both methods that update it (`gather` and `register`) throw an error if they see a duplicate.